LocalWiki.php

```diff
@@ -292,6 +292,12 @@ function onBeforeInitialize( Title &$title, $unused, OutputPage $output, User $u
 			}
 		}
 
+		break;
+	case 'sagan4wiki':
```

Comment:
2 case statements here will combine them since it is the same thing.

Reply:
?

Reply:
Like https://github.com/miraheze/mw-config/blob/1713daaa1315f4a79cb8cdd7a2b93f77234deebc/LocalWiki.php#L310-L311

Reply:
I didn't even know you could do that in PHP.